src/component/organisms/g-menu.tsx

```diff
@@ -6,29 +7,49 @@ import {
   BLACK0,
   BLACK1,
   BLACK2,
+  BLACK3,
+  GREY1,
   PRIMARY2,
   WHITE0,
   WHITE1,
 } from '../atoms/colors';
+import ChevronDownIcon from '@src/assets/icons/common/chevron/down';
+import ChevronUpIcon from '@src/assets/icons/common/chevron/up';
 
 export const GMENU_WIDTH = '200px';
 const GUIDE_HREF =
   'https://www.notion.so/mugles/688bf587c48f4f61b557a4ead533ebda';
 
+type ChildMenuData = {
+  label: string;
+  href: string;
+};
+
 type MenuData = {
   label: string;
   href: string;
+  menuItems: ChildMenuData[];
```

Comment:
```suggestion
  menuItems?: ChildMenuData[];
```

optional propertyㄹㅗ 바꾼 후 대시보드에선 아예 안 써줘도 될 것 같아요!

Reply:
7057cbc